templates/prometheus.yaml

```diff
@@ -10,11 +10,11 @@ metadata:
     # on the cluster, this hack will trigger re-queue on Addons until one exists.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "0.31.1-1"
-    appversion.kubeaddons.mesosphere.io/prometheus-operator: "0.31.1"
-    appversion.kubeaddons.mesosphere.io/prometheus: "2.9.2"
-    appversion.kubeaddons.mesosphere.io/alertmanager: "0.17.0"
-    appversion.kubeaddons.mesosphere.io/grafana: "6.2.5"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.34.0-1"
+    appversion.kubeaddons.mesosphere.io/prometheus-operator: "0.34.0"
+    appversion.kubeaddons.mesosphere.io/prometheus: "2.14.0"
+    appversion.kubeaddons.mesosphere.io/alertmanager: "0.19.0"
+    appversion.kubeaddons.mesosphere.io/grafana: "6.4.2"
```

Comment:
should we also update the values url on https://github.com/mesosphere/kubeaddons-configs/pull/368/files#diff-848433d061871ec88294b7c52b46b08cR24 ?